crates/re_types/definitions/rerun/blueprint/archetypes/series_line.fbs

```diff
@@ -0,0 +1,21 @@
+include "fbs/attributes.fbs";
+include "rust/attributes.fbs";
+
+include "rerun/components.fbs";
+
+namespace rerun.archetypes;
```

Comment:
would be still nice to move this (and _points) to the reguilar archetype directory. A bit confusing ;)